persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/configuration/TableManipulationConfigurationBuilder.java

```diff
@@ -37,11 +36,11 @@
    }
 
    /**
-    * Repetitive DB operations this are batched according to this parameter. This is an optional parameter, and if it
-    * is not specified it will be defaulted to {@link TableManager#DEFAULT_BATCH_SIZE}.
+    * Deprecated. Please use {@link org.infinispan.configuration.cache.AbstractStoreConfigurationBuilder#maxBatchSize(int)} instead.
     */
+   @Deprecated
    public S batchSize(int batchSize) {
-      attributes.attribute(BATCH_SIZE).set(batchSize);
+      attributes.attribute(AbstractJdbcStoreConfiguration.MAX_BATCH_SIZE).set(batchSize);
```

Comment:
I don't think this is as easy it seems, as `TableManipulationConfigurationBuilder` utilises it's own AttributeSet, not the one inherited from `AbstractStoreConfigurationBuilder`, so we can't just call `maxBatchSize` on the parent. Furthermore, `TableManipulation` does not extend `AbstractStoreConfiguration`.